src/jshint.js

```diff
@@ -2622,7 +2618,7 @@ var JSHINT = (function() {
       // first expression *or* the current group contains multiple expressions)
       if (!isNecessary && (first.left || ret.exprs)) {
         isNecessary =
-          (!isBeginOfExpr(preceeding) && first.lbp < preceeding.lbp) ||
+          (!isBeginOfExpr(preceeding) && first.lbp <= preceeding.lbp) ||
```

Comment:
I initially tried that, but it failed to account for cases where like operators were being used on either side of the grouping operator. Since usages like `x = 1 / (2 / 3)` are valid, I think `<=` is the correct approach here.